lib/SensorFusion/src/App.cpp

```diff
@@ -211,6 +216,26 @@ void App::fatalErrorHandler()
     }
 }
 
+void App::publishSensorFusionPosition()
+{
+    StaticJsonDocument<JSON_DOC_DEFAULT_SIZE> payloadJson;
+    char                                      payloadArray[JSON_DOC_DEFAULT_SIZE];
+
+    /* Write newest Sensor Fusion Data in JSON String */
+    IKalmanFilter::PositionData currentPosition = m_sensorFusion.getLatestPosition();
+    payloadJson["positionX"]                    = currentPosition.currentXPos;
+    payloadJson["positionY"]                    = currentPosition.currentYPos;
+    payloadJson["heading"]                      = currentPosition.currentHeading;
+    (void)serializeJson(payloadJson, payloadArray);
+    String payloadStr(payloadArray);
+    m_mqttClient.publish(TOPIC_NAME_POSITION, false, payloadStr);

```

Comment:
Checking the return value?

Reply:
- Added check of return value of MqttClient::publish
- Checking the return value of serializeJson is not necessary since it is the number of bytes written